server/stream.go

```diff
@@ -5679,6 +5677,17 @@ func (mset *stream) getPublicConsumers() []*consumer {
 	return obs
 }
 
+// 2 minutes plus up to 30s jitter.
+const (
+	defaultStreamCheckInterestStateInterval        = 2 * time.Minute
```

Comment:
Could we use shorter names which is more common in Go vs Java like variable name.s?

Reply:
Have shortened them a bit.